substreams-helper/src/token.rs

```diff
@@ -0,0 +1,25 @@
+use crate::pb::evm_token::v1::Token;
+use num_bigint;
+use substreams::scalar::BigInt;
+use substreams_ethereum::pb::eth as pbeth;
+
+pub fn get_eth_token() -> Option<Token> {
+    let eth_token = Token {
+        address: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE".to_string(),
+        name: "Ethereum".to_string(),
+        symbol: "ETH".to_string(),
+        decimals: 18 as u64,
```

Comment:
Avoid type conversion - probs fine as compiler will optimise away although better to use 18_u64 instead.

Reply:
Looks like it works, but what is this `_u64` part? I can't find anything in the rust docs about using the `_`